eradiate/scenes/spectra/_solar_irradiance.py

```diff
@@ -116,6 +119,17 @@ def eval_mono(self, w: pint.Quantity) -> pint.Quantity:
 
         return irradiance
 
+    def eval_rgb(self) -> pint.Quantity:
+        w_units = ureg(self.data.ssi.w.attrs["units"])
+        wavelengths = ([600, 500, 400] * ureg.nm).m_as(w_units)
+        irradiance = to_quantity(self.data.ssi.interp(w=wavelengths, method="linear"))
```

Comment:
This has a high chance of producing irrelevant data due to the fairly high-frequency features of the solar irradiance spectrum. While implementing an appropriate method to compute the RGB values accurately is straightforward, I understand that it's a bit of work and can be part of a separate issue/PR. However, I'd rather have this guy raise or, at least, issue a warning.

Reply:
What approach would you suggest for the RGB values? Or rather, do you have a reference document or something?